README.md

```diff
@@ -93,15 +93,19 @@ In order to build the language the following are needed:
 The `ref_impl` directory contains the reference implementation parser, type checker, interpreter, and command line runner. In this directory, build and test the Bosque reference implementation with:
 
 ```none
-npm install && npm run-script build && npm test
+npm install --production=false && npm run-script build && npm test
 ```
 
 ### Command Line Execution
 
-The `ref_impl` directory contains a simple command line runner for standalone Bosque (`.bsq`) files. These files must have a single `entrypoint` function called `main()` (see [some examples](ref_impl/src/test/apps)). The code in the file can be parsed, type checked, and executed with:
-
+The `ref_impl` directory contains a simple command line runner for standalone Bosque (`.bsq`) files. These files must have a single `entrypoint` function called `main()` (see [some examples](ref_impl/src/test/apps)). 
+Firstly you should compile `app_runner.ts`:
+```none
+tsc --downlevelIteration ref_impl/src/test/app_runner.ts

```

Comment:
Compiling like this produces 665 errors for me. When I move into the `ref_impl/` directory and run `tsc --downlevelIteration src/test/app_runner.ts` instead, it fills up the `src` directory with compiled JavaScript files. What's wrong with using the existing `tsconfig.json` to build the project which specifies the `outDir`? Just running `npm build` does this for you!

Reply:
Oh, thank you) Of course. I'm dumb) I should close this pr))